### DISCUSSION TITLE
There is support like https://freesql.net/ Is there a functional plan for AOT?

### DISCUSSION THREAD

**Discussion by songlinshu** [diesel-rs/diesel#4279] [Q&A]
HI ,There is support like https://freesql.net/ Is there a functional plan for AOT?

**Comment by weiznich** [comment]
I converted your question to a support question as it's no bug report. Please do not fill such questions as bug reports again, we will close future questions without comments.

That said: I'm really not sure what you are asking for. Try to describe your problem with more details if you want to have an answer.
